src/main/web_src/src/ducks/bestillingStatus/bestillingStatusMapper.js

```diff
@@ -65,7 +68,7 @@ const extractBestillingstatusKode = (bestilling, harAvvik, antallIdenterOpprette
 const extractValuesForBestillingListe = (data, statusKode) => {
 	const values = {
 		id: data.id.toString(),
-		antallIdenter: data.antallIdenter.toString(),
+		antallIdenter: data.antallIdenter ? data.antallIdenter.toString() : 0,
```

Comment:
Fungerer denne på organisasjoner også nå? Da bør vi kanskje endre navn fra antallIdenter 😄

Reply:
Njæ, om jeg husker rett var det vel mest en hack for å få org-bestilling til å passe inn i ident-bestilling 😝  Men bør egentlig ikke være sånn - jeg lager en sak 😄